src/index.html

```diff
@@ -4,10 +4,190 @@
+        <p class="text">Tech you can't feel. Leaving just space for sound to come to life. Powered by custom sound technology.</p>
+        <p class="text--price">$ 119.00</p>
+      </section>
+      <h2 class="title title--secondary">Browse Nothing products by category</h2>
+      <div class="gallery">
+        <img src="./images/gallery-image-phone.jpg" alt="Gallery photo phone">
+        <img src="./images/gallery-image-earphones.jpg" alt="Gallery photo earphones">
+        <p class="text--gallery">All products</p>
+      </div>
+      <div class="gallery">
+        <img src="./images/gallery-image-earstick-single.jpg" alt="Gallery photo single earstick">
+        <img src="./images/gallery-image-earstick.jpg" alt="Gallery photo earsticks">
+        <p class="text--gallery">Audio</p>
+      </div>
+      <div class="gallery">
+        <img src="./images/gallery-image-phone2.jpg" alt="Gallery photo phone">
+        <img src="./images/gallery-image-charger.jpg" alt="Gallery photo charger">
+        <p class="text--gallery">Accessories</p>
+      </div>
+      <button class="button"><p class="text--button">View all</p></button>
+    </main>
+    <img src="./images/footer-image.jpg" alt="footer img unknown">
+    <footer class="footer">
+      <h2 class="title title--secondary">About us</h2>
+      <p class="text">We’re a London-based tech company on a mission to remove barriers between people and technology.
+        <br>
+        <br>
+      To do this, we’re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+      No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products we’re proud to share with our friends and family. Simple.
+        <br>
+        <br>
+      Because beautiful tech shouldn’t be complicated.
+      </p>
+      <div class="contact">
+        <h2 class="title title--secondary">Contact us</h2>
+        <form class="form">
+          <div class="form__item">
+            <label
+              for="name"
+              class="form__label"
+            >
+              Name
+            </label>
+            <input
+            id="name"
+            type="text"
+            placeholder="Name"
+            class="form__input"
+            >
+          </div>
+          <div class="form__item">
+            <label
+              for="email"
+              class="form__label"
+            >
+            Email
+            </label>
+            <input
+            id="email"
+            type="text"
+            placeholder="Email"
+            class="form__input"
+            >
+          </div>
+          <div class="form__item">
+            <label
+             for="message"
+             class="form__label"
+            >
+            Message
+            </label>
+            <textarea
+             id="message"
+             placeholder="Message"
+             class="form__textarea"
+             ></textarea>
```

Comment:
add required not to allow sending empty form
fix formatting
you can use self-closing tag